src/assets/scss/homepage.scss

```diff
@@ -280,3 +280,44 @@
 		display: block;
 	}
 }
+
+.deprecation-notice {
+	font-size: 14px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	padding: 2px 0;
+	background-color: var(--link-color);
+	color: var(--body-background-color);
+
+	a {
+		color: var(--body-background-color);
+		padding: 0 4px;
```

Comment:
I don't think you would need padding to anchor element if you were using `<p>` tag for content? It would preserve white space for you?

Reply:
Yes, we can use the `p` tag and remove the margin. However, in the mobile view, the text jumps to a new line because the p tag uses the `display: block` property.